core/src/main/resources/cosmosdb_stored_procedure/mutate.js

```diff
@@ -15,7 +15,9 @@ function mutate(num, ...args) {
 
```

Comment:
One pain point of this change is that we need to change the stored procedure to treat NULL values in Cosmos DB. But if we don't use NULL values, we don't need to update the stored procedure. So, we preserve the backward compatibility, and we can upgrade to a version including this change without any changes. But if we want to use NULL values after upgrading, we need to update the stored procedure.